dapps/assets/src/components/Assets.ts

```diff
@@ -30,6 +30,19 @@ declare module 'vue/types/vue' {
     $router: VueRouter;
     $route: Route;
     $store: any;
+    $t: Function;
+    dapp: Dapp;
+    getRuntime: Function;
+  }
+
+  interface Dapp {
+    baseHash: string;
+    baseUrl: string;
+    contractAddress: string;
+    domainName: string;
+    ens: string;
+    fullUrl: string;
+    rootEns: string;
   }
 }
```

Comment:
This will work once we fix the TS-modules. Replay to Karl:

> Normally this would be done in the component.ts and be available to all EvanComponents. Problem is currently modules arent exported properly so compiler doesn't understand. This should be fixed with TS-fixing instead.